code/datums/traits/negative.dm

```diff
@@ -370,3 +370,20 @@
 		if(prob(1))
 			new/obj/item/reagent_containers/food/snacks/spaghetti/pastatomato(get_turf(H)) //now that's what I call spaghetti code
 
+/datum/quirk/replicant_eat_that
+	name = "Replicant Snob"
+	desc = "You find yourself greatly disliking anything made by a replicator. It's the Old Fashion way, or no way!"
+	value = -1
+	gain_text = "<span class='notice'>You crave something real.</span>"
```

Comment:
"You crave something real." is a very vague statement. Out of context, it would be confusing to the player. Perhaps "You crave some real food." ?